core/pkg/opaprocessor/processorhandler.go

```diff
@@ -44,6 +44,7 @@ func NewOPAProcessor(sessionObj *cautils.OPASessionObj, regoDependenciesData *re
 func (opap *OPAProcessor) ProcessRulesListenner() error {
 
 	policies := ConvertFrameworksToPolicies(opap.Policies, cautils.BuildNumber)
+	opap.OPASessionObj.AllPolicies = policies
```

Comment:
I think we should return the object directly to `opap.OPASessionObj.AllPolicies` to avoid the need of having the garbage collector cleaning up the object

Reply:
You mean changing it to one-line ? It returns a pointer anyways...